Content.Server/Paper/PaperComponent.cs

```diff
@@ -11,7 +11,7 @@ public sealed class PaperComponent : SharedPaperComponent
         public string Content { get; set; } = "";
 
         [DataField("contentSize")]
-        public int ContentSize { get; set; } = 1000;
+        public int ContentSize { get; set; } = 8000;
```

Comment:
How does 8000 characters go with networking?

Reply:
This I'm really not qualified to answer with confidence, but my thinking in including this is that we're talking about a very infrequent transmission with a size so low that even a 56kbps connection could download it in under a second. Maybe I'm not understanding the way SS14's networking exasperates problems like these?

Reply:
I'm not sure I concur that paper _should_ be so short. If books are supposed to be longer, then we can make those longer, unless that does indeed introduce excessive network overhead. I _have_ run into the 1,000 characters limit before; and with fillable forms and templating now possible, I can only imagine this will be more common. There's also the future of rich formatting to consider, as markup is going to need a limit, too.

Right now, we have less room than two tweets to write whatever it is we'd like to write. I think this is plain insufficient.

Reply:
8k characters is i think fine for an increase

Reply:
I think around 5k-6k for paper and like 10-12k for books would be a sweet spot, but you are gonna need to make the books bigger either way @Samsterious

Reply:
All right, I've added 9d76a552fa6489c27de66e02a62a4b2d84b50589 to the PR, implementing those maximums.

Reply:
I think currently admin logs don't track paper but if they ever do in the future then we'll probably need some sort of strategy besides dumping the entire text every time.